icu4j/main/core/src/test/java/com/ibm/icu/dev/test/format/DateTimeGeneratorTest.java

```diff
@@ -2066,4 +2066,35 @@ public void testRegionOverride() {
             assertEquals("Wrong pattern", expectedPattern, actualPattern);
         }
     }
+
+    @Test
+    public void testISO8601More() {
+        final String[][] testCases = {
+            { "en_GB@calendar=iso8601;rg=uszzzz", "EEEEyMMMMdjmm", "y MMMM d, EEEE 'at' h:mm a" },
+            { "en_GB@calendar=iso8601;rg=uszzzz", "EEEEyMMMMdHmm", "y MMMM d, EEEE 'at' HH:mm" },
+            { "en_GB@calendar=iso8601;rg=uszzzz", "Edjmm",         "d, EEE, h:mm a" },
+            { "en_GB@calendar=iso8601;rg=uszzzz", "EdHmm",         "d, EEE, HH:mm" },
+
+            { "en_US@calendar=iso8601",           "EEEEyMMMMdjmm", "y MMMM d, EEEE 'at' h:mm a" },
+            { "en_US@calendar=iso8601",           "EEEEyMMMMdHmm", "y MMMM d, EEEE 'at' HH:mm" },
+            { "en_US@calendar=iso8601",           "Edjmm",         "d, EEE, h:mm a" },
+            { "en_US@calendar=iso8601",           "EdHmm",         "d, EEE, HH:mm" },
+
+            { "en_US",                            "EEEEyMMMMdjmm", "EEEE, MMMM d, y 'at' h:mm a" },
+            { "en_US",                            "EEEEyMMMMdHmm", "EEEE, MMMM d, y 'at' HH:mm" },
+            { "en_US",                            "Edjmm",         "d EEE, h:mm a" },
+            { "en_US",                            "EdHmm",         "d EEE, HH:mm" },
+        };
```

Comment:
This test is actually failing for me on the Java side.  Was there some additional step I (or somebody else) needed to take to get the ISO 8601 changes into the ICU4J data JAR file?

Reply:
Hmmm. I think the goal of this test should be to reflect what is in CLDR right now, and refine later. What are the symptoms of the failures now?

Reply:
Well, I copied the same test from C++ to Java and it passes on C++ and fails on Java, which makes me think my copy of ICU4J didn't pick up the CLDR changes.  That probably means I skipped a step somewhere, but I don't know what I forgot to do.  Or, if this should have happened automatically with the CLDR integration, maybe something was missed?

Reply:
Oops.  It was my own fault.  There was a hack in the actual code that I had forgotten I needed to take out.  Sorry for the extra churn.